Emby.Naming/Video/VideoListResolver.cs

```diff
@@ -175,64 +175,50 @@ private IEnumerable<VideoInfo> GetVideosGroupedByVersion(List<VideoInfo> videos)
                 return videos;
             }
 
+            var list = new List<VideoInfo>();
+
             var folderName = Path.GetFileName(Path.GetDirectoryName(videos[0].Files[0].Path));
-            if (!string.IsNullOrEmpty(folderName))
+
+            if (!string.IsNullOrEmpty(folderName) && folderName.Length > 1)
```

Comment:
Why check for `folderName.Length`?

Reply:
Beats me. I'm scared to change it...

Reply:
Yea, if it's a revert, let's avoid changing what the revert does - that'll get us back to known-working code and can go through the finer points later.